packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -809,7 +853,166 @@ class _CheckboxPainter extends ToggleablePainter {
         }
       }
     }
+    switch (_designSpec!) {
+      case _DesignSpec.cupertino:
+        // Apply effect to darken checkbox when pressed.
+        if (!reaction.isDismissed){
+          final Paint paint = Paint()
+            ..color = Colors.transparent.withOpacity(0.05);
+          final Rect outer = _outerRectAt(origin, 1.0);
+          _drawBox(canvas, outer, paint, activeSide);
+        }
+        if (isFocused) {
+          final Rect focusOuter = _outerRectAt(origin, 1.0).inflate(1);
+          final Paint borderPaint = Paint()
+            ..color = focusColor
+            ..style = PaintingStyle.stroke
+            ..strokeWidth = 3.5;
+          _drawBox(canvas, focusOuter, borderPaint, activeSide);
+        }
+      case _DesignSpec.material:
+        break;
+    }
+  }
+}
+
+// Hand coded defaults for iOS/macOS checkbox
```

Comment:
Period at the end here.

Also, what do you mean by "hand coded" exactly?

Reply:
I will use a better explanatory comment for this. 

What hand coded meant in this context was either eyeballed from checkbox examples in the HIG docs or gotten from `CupertinoCheckbox` which were eyeballed themselves from checkbox examples in the HIG docs.